sale_product_email/tests/test_project.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Commown SCIC SAS (https://commown.fr)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestProjectRatingNPS(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This part should not be here (not related)

Reply:
Indeed, gonna remove this!